src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
+/* eslint-disable */
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function promise1() {
+  logo.addEventListener('click', () =>
+    new Promise((resolve) => resolve('Promise was resolved!')).then(
+      (msg) => (document.body.innerHTML += `<div class="message">${msg}</div>`),
+    ),
+  );
+}
+
+function promise2() {
+  new Promise((resolve, reject) =>
+    setTimeout(reject, 3000, 'Promise was rejected!'),
+  ).catch(
+    (msg) =>
+      (document.body.innerHTML += `<div class="message error-message">${msg}</div>`),
```

Comment:
You're doing almost the same here

Let's create a separate function to add a new massage. As arguments it should accept the message and type (`error` || `success`) - to add the correct class